source/install.md

```diff
@@ -0,0 +1,245 @@
+
+:::{admonition} Existing Conda/Mamba Installations
+If you have an existing Mamba installation, you can use it and skip steps 1 and 2 above. If you have an existing Conda installation, see [](existing_conda).
+:::
+
+:::{admonition} OpenFF on Windows
+We recommend installing MambaForge through WSL on Windows. For more information, see [](install_windows).
+:::
+
+:::{admonition} OpenFF on Apple Silicon
+We recommend installing x86_64 MambaForge on all Apple Macs, even Apple Silicon Macs like the M1 and M2. For more information, see [](install_arm).
+:::
+
+[new environment]: managing_environments
+[projects list]: projects
+[project docs]: projects
+
+(install_mamba)=
+## Installing Mamba
+
+Mamba is much faster than the Conda package manager and can sometimes find ways to safely install two pieces of software that Conda thinks have conflicting dependencies. MambaForge includes Mamba itself, as well as an initial configuration supporting Conda Forge.
+
+If you don't have Conda or Mamba installed, installing MambaForge will give you access to everything you need to install OpenFF software. If you can, we recommend installing MambaForge locally to your user account, rather than system-wide, so that you can freely create and destroy environments and manage your own configuration. If something goes wrong, you can always delete your MambaForge installation and start again, and you'll only lose your installed software.
+
+:::{hint}
+If you're running Mamba through MambaForge, you're configured to use the Conda Forge channel by default, and you don't need to pass the `-c conda-forge` argument to the commands in this page. Including this argument does no harm, so we've included it for the benefit of users that don't have this configuration.
+:::
+
+[Mamba]: https://mamba.readthedocs.io/en/latest/index.html
+[MambaForge]: https://github.com/conda-forge/miniforge#mambaforge
+[MambaForge repository]: https://github.com/conda-forge/miniforge#mambaforge
+[Mambaforge-MacOSX-x86_64]: https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-x86_64.sh
+[Mambaforge-Linux-x86_64]: https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+
+(conda_init)=
+### To init or not to init
+
+After it's done installing, the MambaForge installer will ask you if you want it to initialize MambaForge. Initialization involves adding a section to your shell startup script that tells the shell where to find Conda/Mamba. Answering "no" will leave you startup scripts untouched, but you will need to run an additional command every time you want to use Conda/Mamba or a Conda environment. Answering "yes" will attempt to add a version of this command to your shell startup script, which generally makes Conda easier to use.
+
+If you do answer "yes", we recommend preventing Conda from activating the base environment in every new shell:
+
+```shell-session
+$ conda config --set auto_activate_base false
+```
+
```

Comment:
I've been burned a couple of times by systems failing to update because a package manager was expecting the system Python and got the base Conda environment instead. I know most package managers are more resilient than that but I really don't want to create weird problems for our users.